timber/src/commonMain/kotlin/timber/log/Timber.kt

```diff
@@ -0,0 +1,122 @@
+package timber.log
+
+import org.jetbrains.annotations.NonNls
+
+/** Logging for lazy people. */
+expect class Timber private constructor() {
+
+    /** A facade for handling logging calls. Install instances via [`Timber.plant()`][.plant]. */
+    abstract class Tree {
```

Comment:
This file needs to use 2-space indents. I can add an .editorconfig and get ktlint set up to automatically format.